iceoryx2-bb/posix/tests/semaphore_tests.rs

```diff
@@ -418,46 +417,21 @@ fn unnamed_semaphore_multiple_ipc_semaphores_are_working() {
 }
 
 #[test]
+#[should_panic]
 fn unnamed_semaphore_acquire_uninitialized_ipc_handle_failes() {
     let handle = UnnamedSemaphoreHandle::new();
 
-    let sut = UnnamedSemaphore::from_ipc_handle(&handle);
-    assert_that!(sut, is_err);
-    assert_that!(*sut.as_ref().err().unwrap(), eq AcquireIpcHandleError::Uninitialized);
-
-    let sut1 = UnnamedSemaphoreBuilder::new()
-        .is_interprocess_capable(true)
-        .create(&handle)
-        .unwrap();
-
-    let sut2 = UnnamedSemaphore::from_ipc_handle(&handle);
-    assert_that!(sut2, is_ok);
-
-    drop(sut1);
-
-    let sut3 = UnnamedSemaphore::from_ipc_handle(&handle);
-    assert_that!(sut3, is_ok);
-
-    drop(sut2);
-    drop(sut3);
-
-    let sut = UnnamedSemaphore::from_ipc_handle(&handle);
-    assert_that!(sut, is_err);
-    assert_that!(*sut.as_ref().err().unwrap(), eq AcquireIpcHandleError::Uninitialized);
+    unsafe { UnnamedSemaphore::from_ipc_handle(&handle) };
 }
 
 #[test]
+#[should_panic]
 fn unnamed_semaphore_acquiring_non_ipc_capable_handle_fails() {
     let handle = UnnamedSemaphoreHandle::new();
     let _sut1 = UnnamedSemaphoreBuilder::new()
         .is_interprocess_capable(false)
         .create(&handle)
         .unwrap();
```

Comment:
```suggestion
    let _sut1 = UnnamedSemaphoreBuilder::new()
        .is_interprocess_capable(false)
        .create(&handle);
```

Just to prevent a false positive panic from `unwrap`